src/wallet/components/WalletDropdownBaseName.tsx

```diff
@@ -0,0 +1,63 @@
+import type { Address } from 'viem';
+import { base } from 'viem/chains';
+import { useAccount } from 'wagmi';
+import { useName } from '../../identity/hooks/useName';
+import { Spinner } from '../../internal/components/Spinner';
+import { baseNameSvg } from '../../internal/svg/baseNameSvg';
+import { cn, pressable, text } from '../../styles/theme';
+import type { WalletDropdownBaseNameReact } from '../types';
+
+export function WalletDropdownBaseName({
+  className,
+}: WalletDropdownBaseNameReact) {
+  const account = useAccount();
+
+  const { data: baseName, isLoading } = useName({
+    address: account.address as Address,
+    chain: base,
+  });
+
+  let hasBaseUserName = false;
```

Comment:
Should this be
```
const hasBaseUserName = !!baseName;
```
???

Reply:
It could be. I actually refactored that to what you see now because I thought that `!!` was confusing syntax

Reply:
I feel like what I have now is easier to read.